srcpkgs/falkon/template

```diff
@@ -18,3 +17,14 @@ distfiles="${KDE_SITE}/falkon/${version%.*}/falkon-${version}.tar.xz"
 checksum=ce743cd80c0e2d525a784e29c9b487f73480119b0567f9ce8ef1f44cca527587
 
 LDFLAGS="-Wl,--no-fatal-warnings"
+
+build_options="kde"
+build_options_default="kde"
```

Comment:
put `_default` below `desc`

Reply:
thanks, i didn't think to have multiple values in `vopt_if`